### PR TITLE
New identity

### PR DESCRIPTION
**Pull request by stepansergeevitch**
Added support for a new identity setup. Do not merge

### CODE REVIEW

File: setup.js [hunk 1]
```diff
@@ -44,15 +46,15 @@ function start_db(python_dir, on_success, on_error) {
     [resolve_local_file('scripts/start_database.py'), core.getInput('db_suffix')],
     { env: fb_env }
   );
-  return result.stderr.toString().length == 0 ? on_success(result.stdout.toString().trim('\n'), python_dir) : on_error(result.stderr.toString());
+  return result.status == 0 ? on_success(result.stdout.toString().trim('\n'), python_dir) : on_error(result.stderr.toString());
```
Comment (ptiurin) [inline setup.js:49]: Does this work correctly? I think it returns an object from `spawnSync`? https://nodejs.org/api/child_process.html#child_processspawnsynccommand-args-options

**Comment by ptiurin** [comment]
Otherwise lgtm

File: setup.js (hunk 1 above)
Comment (stepansergeevitch) [reply setup.js:49]: I see status there
```
status [<number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) | [<null>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Null_type) The exit code of the subprocess, or null if the subprocess terminated due to a signal.
```

Comment (stepansergeevitch) [reply setup.js:49]: Using stderr check doesn't work anymore since it's also triggered by warnings (trio warning currently)

**Pull request closed without merging**